### PR TITLE
GIX-1048: Default SNS logo

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Show default logo when SNS project does not provide one.

# Changes

* Add an svg in the static assets folder.
* Use the new svg as default when the logo is not present in the SNS.

# Tests

Not applicable.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Project with default logo:

Screenshot 2022-10-21 at 11 23 55
Screenshot 2022-10-21 at 11 24 02

File: frontend/src/lib/utils/sns.utils.ts [hunk 1]
```diff
@@ -81,8 +81,7 @@ const mapOptionalMetadata = ({
   }
 
   return {
-    // TODO: Use default logo if logo is nullish https://dfinity.atlassian.net/browse/GIX-1048
-    logo: nullishLogo ?? "",
+    logo: nullishLogo ?? "/assets/sns-logo-default.svg",
```
Comment (peterpeterparker) [inline frontend/src/lib/utils/sns.utils.ts:84]: maybe worth adding a constant because this code finds place in a middle of `.utils`?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

File: frontend/src/lib/utils/sns.utils.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/utils/sns.utils.ts:84]: Yes, good point.

**Pull request merged**